google/cloud/spanner_v1/_helpers.py

```diff
@@ -198,7 +208,7 @@ def _make_list_value_pbs(values):
     return [_make_list_value_pb(row) for row in values]
 
 
-def _parse_value_pb(value_pb, field_type):
+def _parse_value_pb(value_pb, field_type, field_name, column_info=None):
```

Comment:
Assign default value to field_name

Reply:
@asthamohta 
`field_name` is **not** an optional parameter and it is similar to `field_type`. The `field_name` is derived from the same dict as that of `field_type`. You can refer it [here](https://github.com/googleapis/python-spanner/pull/909/files#diff-5fcb4175b487a4a9418b28ee746c4397a8862ded6389818dd5c501c608ee8504R102-R103).
I guess this method is internal by the naming convention and hence it would not be a breaking change.
Just want to recheck if this method is public or used externally elsewhere by users? If yes then we can rethink.